Application/EdFi.Ods.Standard/Artifacts/MsSql/Structure/Ods/1301-CreateEdOrgToEdOrgTriggers.sql

```diff
@@ -0,0 +1,24 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TRIGGER edfi.edfi_EducationOrganization_TR_Insert ON edfi.EducationOrganization AFTER INSERT AS
```

Comment:
The schema has to match the target table:
```
Message: Cannot create trigger 'auth.edfi_EducationOrganization_TR_Insert' because its schema is different from the schema of the target table or view.
```